mmpose/core/evaluation/top_down_eval.py

```diff
@@ -250,6 +250,83 @@ def _taylor(heatmap, coord):
     return coord
 
 
+def post_dark(coords, batch_heatmaps, share_heatmap=True):
+    """DARK post-pocessing.
+
+    Note:
+        batch_size: N
+        num_keypoints: K
+
+    Args:
+        coords (np.ndarray[N, K, 2]): Coords in shape of batchsize*num_kps*2.
+        batch_heatmaps (np.ndarray[N, K, H, W]): batch_heatmaps
+        share_heatmap (bool) : True for bottom_up paradigm where all persons
+                    share the same heatmap while False for top_down paradigm
+                    where each person has its own heatmaps.
+
+    Returns:
+        res (np.ndarray): Refined coords in shape of batchsize*num_kps*2.
+    """
+    if not isinstance(batch_heatmaps, np.ndarray):
+        batch_heatmaps = batch_heatmaps.cpu().numpy()
+    batch, num_kps, h, w = batch_heatmaps.shape
+    persons = coords.shape[0]
+    if share_heatmap:
+        assert batch == 1
+    else:
+        assert batch == persons
+    for i in range(batch):
+        for j in range(num_kps):
+            mapij = batch_heatmaps[0, j, :, :]
```

Comment:
double check you want `batch_heatmaps[0, j, :, :]` rather than `batch_heatmaps[i, j, :, :]`

Reply:
batch_heatmaps[i, j, :, :] is better